motioneye/static/js/main.js

```diff
@@ -1414,9 +1414,15 @@ function openSettings(cameraId) {
     }
 
     $('div.settings').addClass('open').removeClass('closed');
-    getPageContainer().addClass('stretched');
+    var pageContainer = getPageContainer();
+    pageContainer.addClass('stretched');
     $('div.settings-top-bar').addClass('open').removeClass('closed');
 
+    if (isSingleView()) {
+        pageContainer.addClass('single-cam-edit');
+        $('div.header').removeClass('single-cam');
```

Comment:
Why is the header single CAM padding class removed here (and re-added below when exiting settings)?

Reply:
Well, buttons are located in the header. If it is hidden, apply button will be too.

Reply:
Ah, makes sense then 😄. I should really test it before commenting.